splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -250,6 +263,15 @@ private void configureHandler() throws Exception {
                 .to(direct("success"));
     }
 
+    protected SSLContextParameters getSslContextParameters() {
```

Comment:
Can you find some better name that would represent the fact that this is trusting anything? Please note, that in a normal situation, this should not be used.